app/routes.py

```diff
@@ -1,2 +1,46 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, request, make_response
+from app import db
+from app.models.Planet import Planet
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["POST","GET"], strict_slashes=False)
+def create_planet():
+    if request.method == "POST":
+        request_body = request.get_json()
+        new_planet = Planet(name=request_body['name'], description=request_body['description'], biggest_moon=request_body['biggest_moon'])
```

Comment:
I suggest doing some input validation here.  Verify that the request body fields either exist and are valid.